common/src/chain/transaction/signature/tests/sign_and_mutate.rs

```diff
@@ -1088,6 +1088,7 @@ fn mutate_output(_rng: &mut impl Rng, tx: &SignedTransactionWithUtxo) -> SignedT
         TxOutput::IssueFungibleToken(_) => unreachable!(), // TODO: come back to this later
         TxOutput::IssueNft(_, _, _) => unreachable!(),     // TODO: come back to this later
         TxOutput::DataDeposit(_) => unreachable!(),
+        TxOutput::AnyoneCanTake(_) => unreachable!(),
```

Comment:
Please check if we have tests that check mutating account spendings/commands

Reply:
yes we do